soc/arm64/xenvm/Kconfig.soc

```diff
@@ -8,9 +8,3 @@ config SOC_XENVM
 	select GIC_V3 if XENVM_USE_GIC_V3
 	select GIC_V2 if !XENVM_USE_GIC_V3
 	select CPU_CORTEX_A72
-
-config XEN_INITIAL_DOMAIN
```

Comment:
This commit will broke build, please add only atomic changes. All platforms (xenvm, xenvm-qemu should be buildable)

Reply:
done